### ISSUE TITLE
Consistent Focus Colors for Keyboard Users

### ISSUE THREAD

**Issue by bertybot** [bertybot/AshleyDesignSystem#13]
**Is your feature request related to a problem? Please describe.**
Focus colors for Keyboard support is wildly inconsistent.  Some places it is black some places it is orange. This could be confusing to a keyboard user since it will be unclear to them when something is actually focused. 

EX: 
1. Go to [Accordion Component](https://storybook.dev.ashleyretail.com/?path=/story/atoms-accordion--accordion-default)
2. Hit Tab to Focus
3. Outline is Black

![image](https://user-images.githubusercontent.com/44912991/191761867-36058127-6cd8-4eb4-8911-014a10ad76ec.png)

1. Go to [Input component](https://storybook.dev.ashleyretail.com/?path=/story/atoms-input-stories-input--basic)
2. hit tab to focus 
3. Outline is orange.

![image](https://user-images.githubusercontent.com/44912991/191761811-6bf15045-08c2-4257-ab67-f3e25f3e8a1f.png)

**Describe the solution you'd like**
Overall I think that the focus colors should be consistent. This will make it clearer to the user when they have focused something. I think the Orange is my preferred since focus colors should stand out, and It adds a good bit of extra character and color to the site when interacting with it. However, black is fine too just as long as they are consistent.

**Describe alternatives you've considered**
We could also try overlays for focus like how they do in [Material](https://mui.com/material-ui/react-button/) but, this will require some extra development to implement.
